roles/keystone-defaults/defaults/main.yml

```diff
@@ -83,6 +83,7 @@ keystone:
     sensu_checks:
       check_keystone_api:
         criticality: 'critical'
+    cron_bin_path:  "/sbin:/usr/sbin:{{ (os == 'rhel' ) | ternary('/usr/bin/', '/usr/local/bin/') }}"
```

Comment:
This shouldn't be needed, and PATH in the template really isn't needed either. On RHEL keystone-manage is in /usr/bin, on ubuntu it's in /usr/local/bin. On both systems those directories are included in $PATH, so we don't need to modify PATH IMO.